code/scripting/api/libs/ui.cpp

```diff
@@ -390,5 +392,10 @@ ADE_FUNC(getBriefing,
 	return ade_set_args(L, "o", l_CmdBrief.Set(Cmd_briefs[0]));
 }
 
+ADE_FUNC(getBriefingMusicName, l_UserInterface_CmdBrief, nullptr, "Gets the file name of the music file to play for the briefing.", "string", "The file name or empty if no music")
+{
+	return ade_set_args(L, "s", common_music_get_filename(SCORE_BRIEFING).c_str());
+}
+
```

Comment:
What about making this a virtvar, or adding a corresponding setBriefingMusic?  I'm assuming there's an opportunity for scripts to run after a mission is loaded but before the briefing starts.

Reply:
This API is specifically meant for the libRocket API and as such I try to keep is as minimal as possible. It is also not guaranteed to be backwards compatible since I would like to replace this with an actual Model-View architecture at some point.

Reply:
Ok.